plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -225,7 +225,8 @@ private ColumnFamilyDescriptor createColumnDescriptor(
             .setTtl(0)
             .setCompressionType(CompressionType.LZ4_COMPRESSION)
             .setTableFormatConfig(basedTableConfig)
-            .setLevelCompactionDynamicLevelBytes(dynamicLevelBytes);
+            .setLevelCompactionDynamicLevelBytes(dynamicLevelBytes)
+            .setDisableAutoCompactions(true);
```

Comment:
Do you want to disable dynamic compaction on all the column families ? How compaction will happen in this case ?